ee/tabby-ui/lib/tabby/auth.tsx

```diff
@@ -271,7 +254,6 @@ function useSignOut(): () => Promise<void> {
 }
 
 interface User {
```

Comment:
nit: rename this to something like JWTPayload

Reply:
Used `IJwtPayload` to differentiate from the `jwtPayload` exported by `jwt-decode`.